app/src/main/res/values/array.xml

```diff
@@ -54,6 +54,7 @@
         <item>@string/player_settings_play_in_app</item>
         <item>@string/player_settings_play_in_vlc</item>
         <item>@string/player_settings_play_in_mpv</item>
+        <item>@string/player_settings_play_in_mpv</item>
```

Comment:
I think this should be something different if you have both installed, unless you can't possibly have both?

Reply:
This should actually be `player_settings_play_in_mpvytdl` to match above file.